edge-util/test/Microsoft.Azure.Devices.Edge.Storage.RocksDb.Test/disk/DiskSpaceCheckerTest.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage.RocksDb.Test.Disk
+{
+    using System;
+    using System.IO;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Storage.RocksDb.Disk;
+    using Microsoft.Azure.Devices.Edge.Util.Test.Common;
+    using Xunit;
+
+    [Unit]
+    public class DiskSpaceCheckerTest
+    {
+        [Fact]
```

Comment:
Won't these tests kill the 'check in' tests on the build server? I am not sure that there will be access to the drive? Also these are not unit tests, more like integration. (so the [Unit] on the class might not be the right attribute

Reply:
They don't kill the check in tests on the build server, since the dotnet checkin's are passing. I do see the merit in making them integration rather than unit if they will slow everything down.